### DISCUSSION TITLE
table inside action modal

### DISCUSSION THREAD

**Discussion by jdion84** [filamentphp/filament#8897] [Help] [answered]
i know i can use ->form to add forms to action modals, but how about tables?

is there a way i can add a table to an action modal, or do i need to use a custom view for that?

**Comment by leandrocfe** [comment, marked as answer]
I prefer rendering tables on Pages, but you can do this

1 - [adding-a-table-to-a-livewire-component](https://filamentphp.com/docs/3.x/tables/adding-a-table-to-a-livewire-component)
2 - Use the `modalContent` to render a custom view - `Action::make('myAction')->modalContent(view('my-custom-view'))`
3 - Render your livewire component on the view `<div>@livewire('my-livewire-table')</div>`
